src/main/java/duke/UI.java

```diff
@@ -0,0 +1,157 @@
+package duke;
+
+import java.util.Scanner;
+
+/**
+ * Manages user interactions in the Duke class.
+ */
+public class UI {
+
+    //The scanner used to scan user inputs
+    private static Scanner scan = new Scanner(System.in);
+
+    /**
+     * Scans user inputs.
+     * @return The string user input scanned by the scanner.
+     */
+    public static String scan() {
+        return scan.nextLine();
+    }
+
+    /**
+     * Sends a greeting message to the user.
+     */
+    public static void greet() {
+        System.out.println("____________________________________________________________\n" +
```

Comment:
The coding standard for this course states to break before an operator. So perhaps you could put the + on the next line, rather than at the end of the previous. I think doing this would make your code more readable! (Also applies to the rest of the methods in this class)